tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -8978,6 +9003,12 @@ SpirvInstruction *SpirvEmitter::processIntrinsicRcp(const CallExpr *callExpr) {
                                    getValueOne(argType), argId, loc);
 }
 
+SpirvInstruction *
+SpirvEmitter::processIntrinsicReadClock(const CallExpr *callExpr) {
+  auto *scope = doExpr(callExpr->getArg(0));
```

Comment:
Do you think adding `assert()` to check the type of scope is useful?
I am worried that other developers might be confused by the name `scope`.
I think if we check somehow it is an integer, it might be helpful.

Reply:
Done